feat: add ActorPermissionLevel enum

tobice opened the pull request:
This enum is required for the Python client so that it supports the new `forcePermissionLevel` run option.

Full context here: https://github.com/apify/apify-core/pull/22681

vdusek commented on src/apify_shared/consts.py:
```diff
@@ -379,6 +379,29 @@ class RunGeneralAccess(str, Enum):
+    Warning: Make sure you know what you are doing when changing this value!
+    """
+
+    #: Full permission Actors have access to all user data in the account.
+    FULL_PERMISSIONS = 'FULL_PERMISSIONS'
```
```suggestion
    FULL_PERMISSIONS = 'FULL_PERMISSIONS'
    """Full permission Actors have access to all user data in the account."""
```

vdusek commented on src/apify_shared/consts.py:
```diff
@@ -379,6 +379,29 @@ class RunGeneralAccess(str, Enum):
+    #: Full permission Actors have access to all user data in the account.
+    FULL_PERMISSIONS = 'FULL_PERMISSIONS'
+
+    #: Limited permission Actors have access only to specific resources:
+    #: - default storages
+    #: - storages provided via input
+    #: - the current run
+    #: - ...
+    #:
+    #: Broadly speaking, limited permission Actors cannot access any account data not related to the current run.
+    #: For details refer to the Apify documentation.
+    LIMITED_PERMISSIONS = 'LIMITED_PERMISSIONS'
```
```suggestion
    LIMITED_PERMISSIONS = 'LIMITED_PERMISSIONS'
    """Limited permission Actors have access only to specific resources:
        - default storages
        - storages provided via input
        - the current run
        - ...
    Broadly speaking, limited permission Actors cannot access any account data not related to the current run.
    For details refer to the Apify documentation.
    """
```

vdusek reviewed (changes requested):
Could you please use standard docstrings instead of comments? Thanks.

vdusek reviewed (changes requested):
Oh, and also please undo all the `upload-time`/`upload_time` changes. You have some old version of uv. More context here -> https://github.com/astral-sh/uv/pull/13176.

tobice commented:
Used the standard docstrings.... it's super weird to have them _below_ the name :smile: 

Also updated `uv` and reverted the `uv.lock` change... but it still changes the version :thinking: It now corresponds to the version in `pyproject.toml` which sounds legit but then I'm not sure why it hasn't been changed already.

vdusek reviewed (approved):
LGTM
